### PR TITLE
Dev/okiselev/master licence

### PR DESCRIPTION
**Pull request by kiselik**
# Description
Files with licence and TTP were updated according to the last internal guidelines. 

Changes proposed in this pull request:
- separate licence and oneDAL TPP 
- update list of components in oneDAL TPP, fix wording
- update info about TBB in IPP, MKL TPPs

### CODE REVIEW

File: THIRD-PARTY-PROGRAMS-MKL [hunk 1]
```diff
@@ -259,51 +259,215 @@ This computer code material was prepared, in part, as an account of work sponsor
...
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.    
 
+-------------------------------------------------------------
 
+The following third party programs have their own third party programs. These additional third party program files are as follows: 
+    1. Intel� Threading Building Blocks (Intel� TBB): https://github.com/intel/tbb/blob/tbb_2020/third-party-programs.txt
```
Comment (outoftardis) [inline THIRD-PARTY-PROGRAMS-MKL]: Not sure, but I think Intel(R) is the correct way to introduce the registered mark. Applicable in other files/parts of the file as well.

File: THIRD-PARTY-PROGRAMS-IPP [hunk 2]
```diff
@@ -0,0 +1,456 @@
+Intel® Integrated Performance Primitives (Intel® IPP) Third Party Programs File
```
Comment (napetrov) [inline THIRD-PARTY-PROGRAMS-IPP]: what is point of using capital letters in filenames? 
Do we have a guidelines here?

File: THIRD-PARTY-PROGRAMS-MKL (hunk 1 above)
Comment (outoftardis) [reply THIRD-PARTY-PROGRAMS-MKL]: I've checked in our Style Guide. Here's a quote:
> Do not use superscript for the registered trademark symbol.

File: THIRD-PARTY-PROGRAMS-IPP (hunk 2 above)
Comment (outoftardis) [reply THIRD-PARTY-PROGRAMS-IPP]: related question: why these files do not have a .txt extension? Is this because of the new guidelines?

**Comment by napetrov** [comment]
Do we need this change? @kiselik

**Comment by napetrov** [comment]
New request will be created to handle license update

**Pull request closed without merging**